src/test/srcShared/fs.test.ts

```diff
@@ -169,12 +174,12 @@ describe('FileSystem', function () {
             it(`creates folder but uses the "fs" module if in C9: '${p}'`, async function () {
                 sandbox.stub(extensionUtilities, 'isCloud9').returns(true)
                 const mkdirSpy = sandbox.spy(fsPromises, 'mkdir')
-                const dirPath = createTestPath(p)
+                dirPath = createTestPath(p)
 
                 await fsCommon.mkdir(dirPath)
 
                 assert.ok(existsSync(dirPath))
-                assert.ok(mkdirSpy.calledOnce)
+                assert.strictEqual(mkdirSpy.callCount, 1)
```

Comment:
Yeah, `ok()` should be avoided almost always because it just gives less information, and it's _almost_ never needed.